pkg/dump/dump.go

```diff
@@ -57,46 +68,41 @@ func Simple(stream io.Writer, body []byte) error {
 	if err != nil {
 		return dumpBytes(stream, body)
 	}
-	if haveJQ() {
-		return dumpCondensedJQ(stream, body)
+	if isTerminal(stream) {
+		return dumpColorSingleLine(stream, data)
 	}
-	return dumpJSON(stream, data)
+	return dumpMonochromeSingleLine(stream, data)
 }
 
-func dumpBytes(stream io.Writer, data []byte) error {
-	_, err := stream.Write(data)
+func dumpColorSingleLine(stream io.Writer, data interface{}) error {
+	encoder := jsoncolor.NewEncoder(stream)
+	err := encoder.Encode(data)
 	if err != nil {
 		return err
 	}
 	_, err = stream.Write([]byte("\n"))
 	return err
 }
 
-func dumpJQ(stream io.Writer, data []byte) error {
-	// #nosec 204
-	jq := exec.Command("jq", ".")
-	jq.Stdin = bytes.NewReader(data)
-	jq.Stdout = stream
-	jq.Stderr = os.Stderr
-	return jq.Run()
-}
-
-func dumpCondensedJQ(stream io.Writer, data []byte) error {
-	// #nosec 204
-	jq := exec.Command("jq", "-c", ".")
-	jq.Stdin = bytes.NewReader(data)
-	jq.Stdout = stream
-	jq.Stderr = os.Stderr
-	return jq.Run()
-}
-
-func dumpJSON(stream io.Writer, data *ordered.OrderedMap) error {
+func dumpMonochromeSingleLine(stream io.Writer, data interface{}) error {
 	encoder := json.NewEncoder(stream)
-	encoder.SetIndent("", "  ")
 	return encoder.Encode(data)
 }
 
-func haveJQ() bool {
-	_, err := exec.LookPath("jq")
-	return err == nil
+func dumpBytes(stream io.Writer, data []byte) error {
+	_, err := stream.Write(data)
+	if err != nil {
+		return err
+	}
+	_, err = stream.Write([]byte("\n"))
+	return err
+}
+
+func isTerminal(writer io.Writer) bool {
```

Comment:
We already use `github.com/mattn/go-isatty` in another place, let's use one mevhanism for both?

Reply:
That makes sense. I will do it.